src/SprykerEco/Yves/CrefoPay/Theme/default/components/molecules/crefopay-payment-form/crefopay-payment-form.twig

```diff
@@ -43,14 +44,15 @@
     {% endmacro %}
 
     {% block requestForm %}
-        {% import _self as macros %}
-
         <div class="is-hidden">
             {{ form_widget(data.paymentMethodSubForm) }}
         </div>
 
         {% for field in data.fields %}
-            {{ macros.crefopayField(field.name, field.attribute, config.name) }}
+            {% block requestFormField %}
+                {% import _self as macros %}
```

Comment:
Why was the `import` moved here?

Reply:
It didn't work when import `macros` was placed outside the block.